src/exampleTypes/TagsQuery.js

```diff
@@ -11,6 +11,17 @@ import DefaultSelect from '../layout/Select'
 import TagsJoinPicker, { tagToGroupJoin } from './TagsJoinPicker'
 let CheckboxDefault = props => <input type="checkbox" {...props} />
 
+let copyTags = node => {
```

Comment:
We can pass this like we do with add/remove but was not sure we really have to etc